app/src/main/java/bean/pwr/imskamieskiego/MapActivity.java

```diff
@@ -355,5 +355,42 @@ public boolean onMenuItemClick(MenuItem item) {
         });
     }
 
+    @Override
+    public void setChangeFloorButtonCoords(int barHeight, int resumeCounter) {
```

Comment:
Jak rozumiem, jej zadaniem jest przesunięcie przycisku do zmiany pięter. Metoda ta jest implementowana z interfejsu od okna z ustawieniami wyszukiwania trasy. I teraz pytanie. Dlaczego okno (fragment) z ustawieniami wyszukiwania trasy ma wiedzieć cokolwiek o przełączniku piętra na mapie? Czemu miało by wiedzieć, że w ogóle taki komponent istnieje? To trochę tak, jakby mieć w pokoju włącznik światła od kuchni. Tak być nie powinno. To zupełnie nie trzyma się kupy i robi nam spaghetti zależności między klasami.

Reply:
Tutaj o to chodzi, żeby z następnego fragmentu wiedzieć jaką ma wysokość ma okienko nawigacji (barHeight). Jeżeli to wiemy możemy dokładnie przesunąć ten przycisk do zmiany miejsca.